homeassistant/components/owlet/__init__.py

```diff
@@ -0,0 +1,83 @@
+"""
+Support for Owlet baby monitors.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/owlet/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import (CONF_USERNAME, CONF_PASSWORD, CONF_NAME)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.discovery import load_platform
+
+
+REQUIREMENTS = ['pyowlet==1.0.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'owlet'
+
+SENSOR_TYPES = [
+    'oxygen_level',
```

Comment:
We should build this list from a common list that the platforms also use. We can add a const.py module in the owlet package that defines these constants. Then import those as needed to the component (here) and in the platforms.